cmd/sandbox-ce/cmd/config.go

```diff
@@ -0,0 +1,216 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/src-d/go-cli.v0"
+)
+
+const orgName = "smacker"
+const repoName = "superset-compose"
+const defaultConfigName = "__default__"
+const composerConfigFileURL = "https://raw.githubusercontent.com/%s/%s/%s/docker-compose.yml"
+
+type configCmd struct {
+	cli.PlainCommand `name:"config" short-description:"Config"`
+}
+
+type configDownloadCmd struct {
+	Command `name:"download" short-description:"Download"`
+
+	Revision string `short:"r" long:"revision" description:"revision to download" default:"master"`
```

Comment:
just a suggestion (maybe a bit controversial): instead of 2 different flags use only one that can be filled by url or by tag. It's easy to distinguish them in code.

Reply:
Even if it's easy to code it, will it be easy to understand for the user? How would you call an option that accepts either a URL or a tag?

Reply:
as I said it might be a controversial suggestion. In my opinion this:
```
srcd download v1.2.3
srcd download https://sourced.tech/customer/special.yml
```
is better than
```
srcd download --revision v1.2.3
srcd download --url https://sourced.tech/customer/special.yml
```

Reply:
what about the argument name for the help? "version" maybe? as in, download the _version_ in this tag, or the _version_ in this URL...

Reply:
*version* is good

Reply:
Done here https://github.com/src-d/superset-compose/pull/29/files#diff-0ef40ee555d59e69cfef14c4d469b691R19